RecoTracker/MkFitCore/src/PropagationMPlex.cc

```diff
@@ -754,7 +757,7 @@ namespace mkfit {
       const float deltaZ = zout - zin;
       const float alpha = deltaZ * tanT * ipt * kinv;
 
-      if (Config::useTrigApprox) {
+      if constexpr (Config::useTrigApprox) {
```

Comment:
one case use vdt::sincosf
or even this
https://github.com/VinInn/ctest/blob/2f4a984ac57b7e41c9a86c2ec27610ae75e1b4e7/floatPrec/sincosV.cpp#L5

Reply:
Good suggestion, and trying VDT is on my wish list, but it will take a lot more study than this trivial change.